rotkehlchen/tests/unit/decoders/test_aave_v3.py

```diff
@@ -1226,3 +1396,57 @@ def test_aave_v3_withdraw_xdai(database, gnosis_inquirer, gnosis_accounts) -> No
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize('ethereum_accounts', [['0x79A4D7448B110bf926d96b726D113f7d7De6781f']])
+def test_aave_v3_interest_on_transfer(database, ethereum_inquirer, ethereum_accounts) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x84a70d14ef53987e1ee267435492100f07a4480b7d8022a8d3508e3a4ee70fbf')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
```

Comment:
do you have a test for withdrawing the 100% of the position?

Reply:
yeah `test_aave_v3_disable_collateral` is that case